### ISSUE TITLE
Open graph meta tags in /reply/{key}

### ISSUE THREAD

**Issue by howawong** [g0vhk-io/budgetq#14]
After #13 is merged,  we can add meta tags in each detail page 
```html
<meta property="og:url"                content="" />
<meta property="og:type"               content="article" />
<meta property="og:title"              content="" />
<meta property="og:description"        content="" />
<meta property="og:image"              content="" />
```

For `og:image`, there will be a separate issue in image generation.
